libs/geoservices/covid/src/lib/pages/maps/components/lockdown-map/lockdown-map.component.ts

```diff
@@ -77,52 +78,41 @@ export class LockdownMapComponent implements OnInit {
       );
 
       map.on('mousemove', 'counties-lockdowns', (e) => {
-        const feature = e.features[0];
+        const [feature] = e.features;
+        const props = feature.properties as County;
         this.countyBoxModel.next({
-          county: feature.properties.NAME
+          county: props.NAME
         });
       });
 
       map.on('click', 'counties-lockdowns', (e) => {
-        const feature = e.features[0];
-        const hoveredCounty = this.statData.filter((county) => county.fips === feature.properties.fips);
-        const updatedDate: Date = hoveredCounty[0]['lockdownInfo'][0]
-          ? hoveredCounty[0]['lockdownInfo'][0]['updated']
-          : null;
-        const createdDate: Date = hoveredCounty[0]['lockdownInfo'][0]
-          ? hoveredCounty[0]['lockdownInfo'][0]['created']
-          : null;
+        const [feature] = e.features;
+        const props = feature.properties as County;
+        const filteredCounty = this.statData.filter((county) => county.fips === props.fips);
+        const hoveredCounty = filteredCounty[0];
+        const updatedDate: Date = hoveredCounty.lockdownInfo[0] ? hoveredCounty.lockdownInfo[0].updated : null;
+        const createdDate: Date = hoveredCounty.lockdownInfo[0] ? hoveredCounty.lockdownInfo[0].created : null;
         this.infoBoxModel.next({
-          name: feature.properties.NAME,
-          state: hoveredCounty[0]['state'],
-          fips: hoveredCounty[0]['fips'],
-          fipsNum: hoveredCounty[0]['fipsNum'],
-          claims: hoveredCounty[0]['claims'],
-          sites: hoveredCounty[0]['sites'],
-          lockdowns: hoveredCounty[0]['lockdowns'],
-          lockdownInfo: hoveredCounty[0]['lockdownInfo'],
+          name: props.NAME,
+          state: hoveredCounty.state,
+          fips: hoveredCounty.fips,
+          fipsNum: hoveredCounty.fipsNum,
+          claims: hoveredCounty.claims,
+          sites: hoveredCounty.sites,
+          lockdowns: hoveredCounty.lockdowns,
+          lockdownInfo: hoveredCounty.lockdownInfo,
           updated: updatedDate,
           created: createdDate
         });
       });
-
       map.dragRotate.disable();
     });
   }
 
-  public getFipsLockdownData(fipsCode: string) {
-    const lockdownUrl = 'https://nodes.geoservices.tamu.edu/api/covid/lockdowns/active/county/' + fipsCode;
-    const requests = forkJoin([this.http.get(lockdownUrl)]);
-    requests.pipe(take(1)).subscribe(([lockdownData]) => {
-      console.log(lockdownData);
-      return lockdownData;
-    });
-  }
-
   public getStats(): void {
     const statsUrl = 'https://nodes.geoservices.tamu.edu/api/covid/counties/stats';
-    const requests = forkJoin([this.http.get(statsUrl)]);
-    requests.pipe(take(1)).subscribe(([recievedData]) => {
+    const requests = forkJoin([this.http.get<DataRecord[]>(statsUrl)]);
+    requests.subscribe(([recievedData]) => {
```

Comment:
Like the lockdowns request you were making previously which was removed due to it not being used, this URL should not be composed in the component. `CountiesService` from `@tamu-gisc/geoservices/data-access` exposes the `getCountyStats` method which returns the same data.

This would possibly eliminate the need to inject the `HttpClient` from this component. 

Additionally, `forkJoin` is not necessary as you're only making a single subscription. The http request can be subscribed to directly.

```js

this.http.get('url').subscribe((response) => {
   // Result here
});
```

Referenced review comment:

https://github.com/TamuGeoInnovation/Tamu.GeoInnovation.js.monorepo/pull/52#discussion_r442996485

Reply:
resolved